src/clients/c++/examples/perf_client/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
```

Comment:
perf_client isn't just an example anymore, so put it in src/clients/c++/perf_client

Reply:
done